File: Latihan_b/py/FootballTeam.py
class FootballTeam():
    # private
    __namaTim = ''
    __negara = ''
    __tahun = 0
    __pemain = ''

    # konstruktor
    def __init__(self, namaTim = '', negara = '', tahun = 0, pemain = ''):
        self.__namaTim = namaTim
        self.__negara = negara
        self.__tahun = tahun
        self.__pemain = pemain

    # set Nama Tim
    def setNamaTim(self, namaTim):
        self.__namaTim = namaTim
    
    # get nama Tim
    def getNamaTim(self):
        return self.__namaTim

    # set Negara
    def setNegara(self, negara):
        self.__negara = negara
    
    # get Negara
    def getNegara(self):
        return self.__negara

    # set tahun berdiri
    def setTahun(self, tahun):
        self.__tahun = tahun
    
    # get tahun berdiri
    def getTahun(self):
        return self.__tahun

    # set pemain
    def setPemain(self, pemain):
        self.__pemain = pemain
    
    # get pemain
    def getPemain(self):
        return self.__pemain

File: Latihan_b/c++/FootballTeam.cpp
#include <iostream>
#include <string>
using namespace std;

class FootballTeam
{
    private:
        string namaTim;
        string negara;
        int tahun;
        string pemain;
    
    public:
        FootballTeam(){}

        // konstruktor
        FootballTeam(string namaTim, string negara, int tahun, string pemain)
        {
            this->namaTim = namaTim;
            this->negara = negara;
            this->tahun = tahun;
            this->pemain = pemain;
        }

        // set nama namaTim
        void setNamaTim(string namaTim)
        {
            this->namaTim = namaTim;
        }

        // get nama namaTim
        string getNamaTim()
        {
            return this->namaTim;
        }

        // set Negara
        void setNegara(string negara)
        {
            this->negara = negara;
        }

        // get nama namaTim
        string getNegara()
        {
            return this->negara;
        }

        // set Tahun berdiri
        void setTahun(int tahun)
        {
            this->tahun = tahun;
        }

        // get Tahun berdiri
        int getTahun()
        {
            return this->tahun;
        }

        // set Pemain
        void setPemain(string pemain)
        {
            this->pemain = pemain;
        }

        // get Pemain
        string getPemain()
        {
            return this->pemain;
        }

        ~FootballTeam(){};

};
